### ISSUE TITLE
Generated deb package could not install

### ISSUE THREAD

**Issue by niku** [johnhamelink/exrm-deb#9]
I ran into error when I executed `sudo dpkg --install` to generated deb file which using default template.

environment: same as #6 
Here is a log:

``` bash
exrm_deb_sandbox$ sudo dpkg --install rel/exrmdebsandbox-0.0.1.deb 
dpkg: warning: parsing file '/var/lib/dpkg/tmp.ci/control' near line 8 package 'exrmdebsandbox':
 missing description
dpkg: error processing archive rel/exrmdebsandbox-0.0.1.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 16:
 several package info entries found, only one allowed
Errors were encountered while processing:
 rel/exrmdebsandbox-0.0.1.deb
```

**Comment by johnhamelink** [comment]
Hi @niku - can you try the latest git ref?

``` elixir
{:exrm_deb, github: "johnhamelink/exrm_deb", ref: "b478fde"}
```

**Comment by niku** [comment]
It works! Thank you :smiley: 

```
exrm_deb_sandbox$ sudo dpkg --install rel/exrmdebsandbox-0.0.1.deb 
Selecting previously unselected package exrmdebsandbox.
(Reading database ... 48603 files and directories currently installed.)
Preparing to unpack rel/exrmdebsandbox-0.0.1.deb ...
Unpacking exrmdebsandbox (0.0.1) ...
Setting up exrmdebsandbox (0.0.1) ...

exrm_deb_sandbox$ git diff HEAD      
diff --git a/mix.exs b/mix.exs
index ed66bb8..7b617a2 100644
--- a/mix.exs
+++ b/mix.exs
@@ -30,7 +30,7 @@ defmodule ExrmDebSandbox.Mixfile do
   #
   # Type "mix help deps" for more examples and options
   defp deps do
-    [{:exrm_deb, github: "johnhamelink/exrm_deb", ref: "5d09389"}]
+    [{:exrm_deb, github: "johnhamelink/exrm_deb", ref: "b478fde"}]
   end

   defp package do
```

**Comment by johnhamelink** [comment]
Brilliant! I've released this now - you can upgrade to version 0.0.5 to get the bugfixes for #6 and #9.

Thanks for your report!
